CheckSequence.java

```diff
@@ -0,0 +1,22 @@
+/**
+ * Determines whether the sequence is non-decreasing,
+ * and displays the result on the screen.
+ * @param gaps the sequence that was entered by the user

```

Comment:
I've changed the javadoc for a method, but it still shows this comment. I don't understand what is wrong and where is mistake.